AmmoDriver.md

```diff
@@ -0,0 +1,339 @@
+
+#### Activation States
+
+Activation states are only used for `type: dynamic` bodies (. Most bodies should be left at the default `activationState: active` so that they can go to sleep (sleeping bodies are very cheap). It can be useful to set bodies to `activatioState: disable_deactivation` if also using an `ammo-constrant` as constraints will stop functioning if the body goes to sleep, however they should be used sparingly. Each activation state has a color used for wireframe rendering when debug is enabled.
+
+| state                   | debug rendering color | description                                                                                                                                                                                 |
+| ----------------------- | --------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `active`                | white                 | Waking state. Bodies will enter this state if collisions with other bodies occur. This is the default state.                                                                                |
+| `island_sleeping`       | green                 | Sleeping state. Bodies will enter this state if they fall below `linearSleepingThreshold` and `angularSleepingThreshold` and no other `active` or `disable_deactivation` bodies are nearby. |
+| `wants_deactivation`    | cyan                  | Intermediary state between `active` and `island_sleeping`. Bodies will enter this state if they fall below `linearSleepingThreshold` and `angularSleepingThreshold`.                        |
+| `disable_deactivation`  | red                   | Forced `active` state. Bodies set to this state will never enter `island_sleeping` or `wants_deactivation`.                                                                                 |
+| `disable_simulation`    | yellow                | Bodies in this state will be completely ignored by the physics system.                                                                                                                      |
+
+#### Collision Filtering
+
+Collision filtering allows you to control what bodies are allowed to collide with others. For Ammo.js, they are represented as two 32-bit bitmasks, `collisionFilterGroup` and `collisionFilterMask`.
+
+Using collision filtering requires basic understanding of the [bitwise OR](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_Operators#(Bitwise_OR)) (`a | b`) and [bitwise AND](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_Operators#(Bitwise_AND)) (`a & b`) operations.
```

Comment:
I find it much easier to do collision filtering with hexadecimal values. I can't remember if A-Frame's property syntax supports that, but if so we could simplify this a bit.